docs/docs/guides/formatting-your-fields.mdx

```diff
@@ -20,32 +26,32 @@ In your dbt YAML file, it'll look something like this:
 ```yaml
 version: 2
 models:
-- name: users
-  columns:
-    - name: first_name
-      meta:
-        dimension:
-          hidden: true
+    - name: users
```

Comment:
I think this is adding too many spaces in the .yml? Normally it's 2-spaced tabs, not 4-spaced?